ultranest/utils.py

```diff
@@ -441,3 +441,18 @@ def verify_gradient(ndim, transform, loglike, gradient, verbose=False, combinati
             print("expectation was L=", Lexpected, ", given", Lref, grad, eps)
         assert np.allclose(Lprime, Lexpected, atol=0.1 / ndim), \
             (u, uprime, theta, thetaprime, grad, eps * grad / L, L, Lprime, Lexpected)
+
+def todo_points_for_this_process(mpi_rank, num_live_points_missing, mpi_size):
+    """
+    Calculates number of living points to be evaluated by this process.
```

Comment:
Maybe:

Computes the number of tasks for process number `mpi_rank`, so that
`num_total tasks` are spread uniformly among `mpi_size` processes.

Reply:
done